v22.2/cloud-storage-authentication.md

```diff
@@ -201,30 +201,78 @@ You can use workload identities with assume role authentication to run the follo
 
 To use assume role authentication, you will need at least two IAM roles:
 
-- An _identity role_: the IAM role you have associated with your Kubernetes service account.
+- An _identity role_: the IAM role associated with your Kubernetes service account.
 - An _operation role_: the IAM role to be assumed. This contains the permissions required to complete a CockroachDB operation.
 
-### Set up Amazon S3 workload identity
+For a step-by-step on how to set this up for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters, see the following steps.
+
+### Step 1. Set up the identity role
+
+The first step to setting up workload identity authentication is different for {{ site.data.products.core }} and {{ site.data.products.dedicated }} clusters. As you work through the step, ensure you use the relevant section for your cluster:
+
+- [{{ site.data.products.dedicated }}](#set-up-the-identity-role-for-cockroachdb-dedicated-clusters)
+- [{{ site.data.products.core }}](#set-up-the-identity-role-for-cockroachdb-self-hosted-clusters)
+
+#### Set up the identity role for CockroachDB dedicated clusters
+
+Each {{ site.data.products.dedicated }} cluster has a pre-configured IAM role for [Amazon EKS](https://aws.amazon.com/eks/) service accounts that act as the cluster's identity. In addition, the clusters have a unique built-in, functional IAM role, which you can configure as a trusted identity within the Trust Policy of your cloud IAM roles. This allows you to have the built-in IAM role assume another role (or roles as part of a [chain](#amazon-s3-role-chaining)).
```

Comment:
We refer to the functional IAM role as the "functional**ity** IAM role" as to distinguish it from the "workload" IAM role. Admittedly, [it's not the best name](https://docs.google.com/document/d/1Ot7mGQqRVDC0miNarahCcBDArd7DW8WHuc6lz-hr-hQ/edit#heading=h.sp6wa4etd8x3), but I think that ship has sailed.

Reply:
I have altered this mention of "functional" to "functionality" in both the opening paragraphs for AWS and GCP.